### DISCUSSION TITLE
Logic/logic01/

### DISCUSSION THREAD

**Discussion by giscus** [HobbitQia/notebook#5] [General]
# Logic/logic01/

HobbitQia的笔记本

http://note.hobbitqia.cc/Logic/logic01/

**Comment by Yusux** [comment]
- D-to-A: 处理后把数字信号转化为 **膜你** 信号
  - 模拟 | 膜您 ❗
- Given M elements to be reprented by a binary code, the minimum number of bits, n satisfies  $2^{n} \ge M \gt 2^{n-1}$ so $n = \lfloor log_2M \rfloor$.
  - $n = \lceil log_2M \rceil$
- 对奇校验来说，当 1 的个数为奇数时校验位为 1
  - 对奇校验来说，当 1 的个数为奇数时校验位为 0

**Reply by HobbitQia** [reply]
谢谢华子喵，虽然现在才看到也不想改了/鼻血

**Comment by The0xKa1** [comment]
有点无聊，膜一下俊哥
